standard_reports/expiring-items/latest/src/query.graphql

```diff
@@ -11,7 +11,6 @@ query ExpiringItems(
       hasPacksInStore: true
       isActive: true
     }
-    page: { first: 5000 }
```

Comment:
these were added to include as many items as possible - now removing so we actually get all of them